src/js/popup.js

```diff
@@ -178,7 +187,13 @@ function limitAccess (callback = false) {
                     $('#customTemplateIssueTypeField').prop('disabled', true);
                     $('#customTemplateProjectsField').prop('disabled', true);
                     $('#addDefaultDropdownButton').addClass('disabled');
-                    $('#customDomains').addClass('disabled');
+                    $('#customSettings').addClass('disabled');
+                    $('.custom-domain-ui').each(function () {
+                        $(this).addClass('disabled');
+                    });
+                    $('.custom-inputID-ui').each(function () {
```

Comment:
No need to add `disabled` to 'custom-input-id' or 'custom-domain-id' here since we're already disabling access to the page.

Reply:
Nice catch 👍Thanks, I will update it